FirebaseMessaging/Sources/Token/FIRMessagingTokenInfo.m

```diff
@@ -168,9 +168,12 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
                          fromData:rawAPNSInfo
                             error:&error];
     if (error) {
-      FIRMessagingLoggerDebug(
+      FIRMessagingLoggerInfo(
           kFIRMessagingMessageCodeTokenInfoBadAPNSInfo,
-          @"Could not parse raw APNS Info while parsing unarchived token info: %@", error);
+          @"Could not parse raw APNS Info while parsing unarchived token info: %@. If you are "
+          @"still using the deprecated InstanceID SDK to handle FCM registration token. Please "
```

Comment:
nit: I guess, we can detect if IID is actually installed in the app (e.g. like [here](https://github.com/firebase/firebase-ios-sdk/blob/4ac113a271f22e156050cb62261606880080b704/FirebaseInstallations/Source/Library/FIRInstallations.m#L267)) so we can be more precise in our messaging. I'm not too strong about it though.

Reply:
I think we should add this error in general for the breaking change when we remove IID dependency.